persistence/gov.go

```diff
@@ -35,6 +40,34 @@ func (p PostgresContext) InitParams() error {
 	return err
 }
 
+func (p PostgresContext) GetParameter(paramName string, height int64) (v any, err error) {
+	st := reflect.TypeOf(types.Params{}) // Extract type of parameter from the Params struct
+	var typ reflect.Type
```

Comment:
Wdyt of initializing a local in memory map using the `init` function that keeps a mapping of the tags from `persistence_genesis.pb.go` to avoid looping every time GetParam is called.

Reply:
@Olshansk Great idea speeds things up a lot! Implemented as well - removes the need for any RegEx check as well - 1 less potential error.